assets/app/scripts/controllers/projects.js

```diff
@@ -24,6 +24,45 @@ angular.module('openshiftConsole')
       $('#openshift-logo').off('click.projectsPage');
     });
 
+    $scope.toggleDelete = function(project) {
+      // to toggle prompt to delete
+      // disable the delete button if current project is being deleted
+      if (!$scope.projects[project]['deleting']) {
+        $scope.projects[project]['toggled'] = !$scope.projects[project]['toggled'];
+      }
+    };
+
+    $scope.deleteProject = function(project) {
+      $scope.alerts = {};
```

Comment:
only clear the alert related to this particular project, something like `delete $scope.alerts[project.metadata.name];`